lines/producer/sender.py

```diff
@@ -0,0 +1,45 @@
+"""Kafka Publish Modules."""
+import json
+import os
+
+from kafka import KafkaProducer
+from prefect import Task
+
+from lines.libs.errors import MalformedPayloadError
+
+
+class EventProducer(Task):
+    """Event Producer Object."""
+
+    def __init__(self, *args, **kwargs):
+        self._server = os.getenv("KAFKA_BOOTSTRAP_SERVER")  # temporary
+        self._topic = os.getenv("KAFKA_TOPIC")
+        super().__init__(*args, **kwargs)
+
+    def run(self, record: dict):
```

Comment:
Might be better named `send`?

Reply:
Subclasses of `Task` are executed by `run(...)`

Reply:
Yup, totally my bad.